### PR TITLE
Add show printing method to result class, helps for #28

### PR DESCRIPTION
**Pull request by manycoding**
I was thinking about an easier printing feature for rules, and here it is.

### CODE REVIEW

File: src/arche/rules/result.py [hunk 1]
```diff
@@ -120,3 +120,8 @@ def get_stats_count(self):
                 if message.stats is not None
             ]
         )
+
+    def show(self, short: bool = False, keys_limit: int = 10):
+        from arche.report import Report
```
Comment (manycoding) [inline src/arche/rules/result.py]: This thing concerns me, it is a circular dependency from `result.py` to `report.py`, which I temporaly circumvented by importing on a method level.

Comment (ejulio) [reply src/arche/rules/result.py]: What is the specific idea here?
Do you want to show only a specific result?

File: src/arche/report.py [hunk 2]
```diff
@@ -19,51 +19,63 @@ def wipe(self):
     def save(self, result):
         self.results[result.name] = result
 
-    def write_color_text(self, text, color=Fore.RED, style=""):
+    @staticmethod
+    def write_color_text(text, color=Fore.RED, style=""):
         print(color + style + text + Style.RESET_ALL)
 
-    def write_rule_name(self, rule_name):
+    @staticmethod
```
Comment (ejulio) [inline src/arche/report.py]: Maybe all there static `write_something` methods could go into some sort of `utils` module.

Comment (manycoding) [reply src/arche/report.py]: They belong to reporting, be it a module or a class with one simple goal - report something to jupyter output.

File: src/arche/rules/result.py (hunk 1 above)
Comment (manycoding) [reply src/arche/rules/result.py]: Yep, the idea is to simplify using specific checks with chaining - `something_i_want_to_check(input).show()`

**Comment by manycoding** [comment]
I tried different variants and the most simple seems is to leave it as it is.

**Pull request merged**